### ISSUE TITLE
control the smart speaker via android app

### ISSUE THREAD

**Issue by alok760** [fossasia/susi_android#2282]
**Describe the feature you'd like**
### Use Smart Speakers soundserver to control the smart speaker

See: https://github.com/fossasia/susi_installer/tree/development/raspi/soundserver
The soundserver offers a webpage to control the smart speaker
Now there are two options to control the smart speaker via the android app.
* Use web view to open the webpage
* Use the endpoint directly and build GUI natively on the app

**Comment by atm1504** [comment]
@alok760 can you please say how we have the access to the server. Is it only possible when my device is connected to the same wifi with which my mobile is connected?

**Comment by alok760** [comment]
The android app will have to search the smart speaker on the same network, for that we can either use the Raspberry PI's mDNS name or scan the network for the server
Next thing is authentication, if the web view is used it'll handle the authentication, otherwise authentication should also take place if a password is set.

The authentication works like this - 
- By default there is no password set on the smart speaker, opening the control webpage doesn't require password. The endpoints are also open.
- The user has the option to set up a password if needed. If the password is set, a login screen appears before the smart speaker control webpage and all the endpoints only work if the user is authorized

The control webpage looks like this 
<img src="https://user-images.githubusercontent.com/14289201/60256689-dcdcac80-9886-11e9-9cb3-9c9165163e91.png" height="500px">

**Comment by atm1504** [comment]
> The android app will have to search the smart speaker on the same network, for that we can either use the Raspberry PI's mDNS name or scan the network for the server

@alok760 please elaborate this part

**Comment by atm1504** [comment]
To be solved after #2284. Because that issue contains the establishment of Manage Devices option in the setting page like that present in google home.

**Comment by mariobehling** [comment]
Please use the web view. @alok760 Please integrate it into the my devices section on the web client.

**Comment by atm1504** [comment]
Yes, @mariobehling. I have already integrated a webview with a dummy URL in the pr #2299. I will replace with the actual link after I get a working image of susi installer
